holoviews/plotting/renderer.py

```diff
@@ -69,6 +69,17 @@
 </html>
 """
 
+embed_js = """
```

Comment:
I suggest leaving a comment pointing to this PR where we can explain why such a nasty thing exists.

Reply:
Agreed, don't want anyone blaming us for this.